### ISSUE TITLE
Enhancement: Add missing alt text to emoticon

### ISSUE THREAD

**Issue by TheFabulousPika** [TheFabulousPika/steam-chatlog-to-text#54]
Emoticon alt text is missing in the original chat
```
<img src="https://community.cloudflare.steamstatic.com/economy/emoticonlarge/winter2019tiredyul" class="emoticon_emoticon_9hYGN emoticon_large_p8Q42" data-emoticon="winter2019tiredyul" data-copystyle="merge-adjacent" data-copytext=":winter2019tiredyul:">
```
Add feature to extension that adds alt text

**Comment by TheFabulousPika** [comment]
```
function addAltTextToEmoticon(a){
	var thisMsgNode = a;
	var allEmoticons = thisMsgNode.querySelectorAll('[class*=emoticon_emoticon]');
	for (var i = 0; i < allEmoticons.length; i++){
		var thisEmoticon = allEmoticons[i];
		var altText = document.createAttribute("alt");
		altText.value = thisEmoticon.getAttributeNode("data-copytext").value;
		thisEmoticon.setAttributeNode(altText);
	}
	return thisMsgNode;
}
```
